### PR TITLE
Refactor [v125] Remove extra level of focus-ios folder added by migration script

### PR DESCRIPTION
**Pull request by isabelrios**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-2440)

## :bulb: Description

After running the monorepo scripts a focus-ios folder was created where focus-ios repo was added. So, we had an extra level. 
In this PR, I'm taking out focus-ios from that folder and those files that we need in root directory. 
There are some folders like taskcluster, that we are not moving here since the idea would be to share the ones here. Similarly for other files (.gitignore, .github/workflows....)

 

## :pencil: Checklist
You have to check all boxes before merging
- [ ] Filled in the above information (tickets numbers and description of your work)
- [ ] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

File: .gitignore
Comment (lmarceau) [inline .gitignore:132]: Side note, it seems we have a `firefox-ios/shavar-prod-lists` and a `shavar-prod-lists` folder. I think we can improve this (in another task) as Focus iOS is fetching an older version of the Shavar-prod-lists than Firefox is doing. At least we could ensure that the Focus shavar-prod-lists is under the Focus folder.

I'll create a task to update this once this PR is merged

Comment (lmarceau) [reply .gitignore:132]: Created https://mozilla-hub.atlassian.net/browse/FXIOS-8554 for this

File: focus-ios/.github/CODEOWNERS
Comment (lmarceau) [inline focus-ios/.github/CODEOWNERS:22]: I'll review the codeowners at the root of the project to ensure Focus has the right codeowners with https://mozilla-hub.atlassian.net/browse/FXIOS-8555 once this PR is merged

**Review by lmarceau — APPROVED** [review]

**Review by AaronMT — APPROVED** [review]

**Pull request merged**
